src/structures/ClientApplication.js

```diff
@@ -5,41 +5,69 @@ const Application = require('./interfaces/Application');
 
 /**
  * Represents a Client OAuth2 Application.
+ * <info>The only guaranteed properties are `id` and `flags`. To get the entire
+ * application, you need to fetch it with `application.fetch()` first.</info>
  * @extends {Application}
  */
 class ClientApplication extends Application {
   _patch(data) {
     super._patch(data);
 
+    /**
+     * The flags this application has
+     * @type {number}
+     */
+    if ('flags' in data) this.flags = data.flags;
```

Comment:
```suggestion
    if ('flags' in data) {
      /**
       * The flags this application has
       * @type {number}
       */
      this.flags = data.flags;
    }
```
others also need changing, this is so the docstring is parsed correctly